modules/REST/V1/Options.php

```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\REST\V1;
+
+use WP_Error;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+use WP_Syntex\Polylang\Options\Options as Options_Registry;
+use WP_Syntex\Polylang\REST\Abstract_Controller;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Options REST controller.
+ *
+ * @since 3.7
+ */
+class Options extends Abstract_Controller {
```

Comment:
```suggestion
class Settings extends Abstract_Controller {
```
So it matches the namespace of the route.

Reply:
Done in [0fa8170](https://github.com/polylang/polylang/pull/1569/commits/0fa8170a4ab309e3e0c5614ce765762d84587602).